packages/stripe/lib/src/stripe.dart

```diff
@@ -284,6 +284,16 @@ class Stripe {
     }
   }
 
+  Future<PaymentMethod> createPaymentMethodWithElements() =>
+      _platform.createPaymentMethodWithElements();
+
+  Future<void> elementsSubmit() => _platform.elementsSubmit();
+
+  Future<PaymentIntent> handleNextActionForNonCard(
```

Comment:
I am not convinced about the naming of this method it is very confusing for users. Why cant we use handle next action?

Reply:
In web_stripe.dart we already have a 'handleNextAction' method but it is calling js function 'handleCardAction'

```
  @override
  Future<PaymentIntent> handleNextAction(String paymentIntentClientSecret,
      {String? returnURL}) async {
    final stripe_js.PaymentIntentResponse response =
        await _stripe.handleCardAction(paymentIntentClientSecret);

    return response.paymentIntent!.parse();
  }
```

The method handleNextActionForNonCard is calling the js `handleNextAction` method. Now it would be better to rename the initial `handleNextAction` method to `handleCardAction` but that would be a breaking change on existing codebases and I didn't really know the policy of this package regarding that hence why I came up with the new name.